service/src/CpufreqSysfsDriver.cpp

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright (c) 2015 - 2022, Intel Corporation
```

Comment:
```suggestion
 * Copyright (c) 2015 - 2023, Intel Corporation
```

We can update the whole codebase to 2024 later.